tools/ci/Cargo.toml

```
@@ -0,0 +1,10 @@
+[package]
+name = "ci"
+version = "0.1.0"
+authors = ["Nathan Stocks <nathan@agileperception.com>"]
```

```suggestion
authors = [
  "Bevy Contributors <bevyengine@gmail.com>",
  "Carter Anderson <mcanders1@gmail.com>",
]
```
The authors entry should be the same as all other crates in the Repo.

---

Good point. I updated the `authors` entry as per [the pattern for wholly-contributed crates](https://github.com/bevyengine/bevy/blob/b71ada03465a0343b4ead1b2896f44587e719a47/crates/bevy_tasks/Cargo.toml#L5-L9) that I see.